LearnNet_CartingService/Core/Interfaces/ICartRepository.cs

```diff
@@ -0,0 +1,13 @@
+﻿using LearnNet_CartingService.Domain.Entities;
+
+namespace LearnNet_CartingService.Core.Interfaces
+{
+	public interface ICartRepository
```

Comment:
why do you need to return bool? (also applies to ICartService)

Reply:
trivial success or not result